### ISSUE TITLE
`There was a problem loading the map data` error, even if imagery set to `none`

### ISSUE THREAD

**Issue by jonpsp** [systemed/potlatch3#43]
When I used Potlatch 3 today, I got errors like the following.  When I had the background set to some kind of imagery, not all the tiles would load.  Whether I had the imagery set to something or `none`, the map data would not load for the first few (or more) attempts - eventually bits would load.

```
There was a problem loading the map data.
Please check your internet connection, or try zooming in.

https://www.openstreetmap.org/api/0.6/map?bbox=%2D0%2E9007336646318437%2C51%2E16294176506863%2C%2D0%2E8950983434915544%2C51%2E165374763158376
```

**Comment by jonpsp** [comment]
Annoyingly, the first time I tried to save my changeset, I got an unhelpful error (I can't remember what it said, just something like "Error creating changeset").  I saved it again with a new changeset comment without making any more changes, and it saved.  Here's the version details:-
![image](https://github.com/systemed/potlatch3/assets/20799382/4f4ada3c-1427-46e0-8642-e424c003090e)

I compared the file hash of the current version on the download page with the installer I downloaded - they match.

I've use Potlatch successfully in the past, but it has been months since I last used it before today.
